.github/workflows/ci.yml

```
@@ -238,13 +238,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          override: true
```

```suggestion
      - uses: dtolnay/rust-toolchain@stable
```

Not sure that change is needed in this PR, but we don't use `actions-rs/toolchain` anymore